packages/ember-toucan-form/CHANGELOG.md

```diff
@@ -1,5 +1,28 @@
 # @crowdstrike/ember-toucan-form
 
+## 0.4.0
+
+### Minor Changes
+
+- [#254](https://github.com/CrowdStrike/ember-toucan-core/pull/254) [`532ab89`](https://github.com/CrowdStrike/ember-toucan-core/commit/532ab8997917c9dc4beb3da93e19578ec73b3f09) Thanks [@clintcs](https://github.com/clintcs)! - Add Multiselect test helpers.
```

Comment:
Question: why is adding multiselect test helpers a `minor` change but adding `Autocomplete` test helpers a `patch` change below?